src/frontend/src/components/Searching.tsx

```diff
@@ -41,17 +49,20 @@ export default function Searching() {
       if (e.key === 'Enter') {
          e.preventDefault();
          setPress(true);
-         handlePost;
-         //setAmountFound(apartments.length);
+         handlePost(query, selected);
       }
    };
 
-   function handlePost(query: string) {
+   function handlePost(query: string, select: string[]) {
       axios({
-         method: 'post',
-         url: 'http://localhost:3333/mockdata', // need mainpage.py url
+         method: 'POST',
```

Comment:
This should be a get request, we are not posting stuff to the server.

Reply:
How would the query get to the flask backend? The same thing happens in `Login.tsx`/`Register.tsx`.

Reply:
If you mean getting all the apartments when the page loads, that would make the infinite scroll pretty hard

Reply:
We send a get request that contains some amount of apartments. The backend methods already took this into account.

> If you mean getting all the apartments when the page loads, that would make the infinite scroll pretty hard

Reply:
> How would the query get to the flask backend? The same thing happens in `Login.tsx`/`Register.tsx`. The flask backend then does a get request in the /home url.

We add a query string to the get request.

Reply:
https://stackabuse.com/get-request-query-parameters-with-flask/
This discusses how the backend would handle the query strings.

Reply:
So I should do this thing [https://bobbyhadz.com/blog/react-update-query-params](https://bobbyhadz.com/blog/react-update-query-params)?